### ISSUE TITLE
Some machines aren't working

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#12376]
A number of machines failed Tues night, and I see there are still problems Wed night. Looking at https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.system_ppc64le_linux_Nightly_testList_1/97/console the job appears hung. I'm taking the bad machines offline.

**Comment by pshipton** [comment]
@jdekonin

**Comment by jdekonin** [comment]
I only see cent6-x64-5 and ub16p8j94 disabled.  I've rebooted them both and enabled on jenkins.

**Comment by pshipton** [comment]
Yes, there were only two. More concerning are the other failures that have occurred the past two nights. Not sure if there is anything we can do, or if there are just intermittent network issues.

**Last night:**

https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.system_ppc64le_linux_Nightly_testList_1/97/ - ub16p8j94
https://ci.eclipse.org/openj9/job/Test_openjdk16_j9_sanity.system_ppc64le_linux_Nightly_testList_2/39/ - cent7-ppcle-6

https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_special.system_x86-64_linux_Nightly_testList_0/96/ - cent6-x64-4

**Previous night:**

https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.system_x86-64_linux_Nightly_testList_0/96 - cent6-x64-7
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_special.system_ppc64le_linux_Nightly_testList_0/89/ - no machine
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_special.system_ppc64le_linux_Nightly_testList_3/89/ - cent7-ppcle-3
https://ci.eclipse.org/openj9/job/Test_openjdk16_j9_sanity.system_ppc64le_linux_Nightly_testList_1/59/ - cent7-ppcle-4

https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_sanity.system_x86-64_linux_Nightly/638 - cent6-x64-7 
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.system_x86-64_linux_Nightly_testList_1/87/ - cent6-x64-2

https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.system_x86-64_mac_Nightly_testList_1/89/ - osx1011-x86-1

**Comment by pshipton** [comment]
**April 8:**

https://ci.eclipse.org/openj9/job/Build_JDK8_x86-64_linux_Nightly/687/ - cent6-x64-5

**Comment by pshipton** [comment]
I see ub16x64j92 and ub16x64j94 are offline.
